integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/dataretention/DataRetentionTestCase.scala

```diff
@@ -132,7 +132,15 @@ class DataRetentionTestCase extends QueryTest with BeforeAndAfterAll {
     // delete segments (0,1) which contains ind, aus
     sql(
       "DELETE SEGMENTS FROM TABLE DataRetentionTable where STARTTIME before '" + actualValue + "'")
+    checkAnswer(
```

Comment:
revert this change, as test execution order shouldnot impact testcase behaviour.